lib/response/get/index.js

```diff
@@ -19,6 +19,6 @@ module.exports = function get() {
         });
     }
 
-    var paths = pathSyntax.fromPathsOrPathValues(arguments);
+    var paths = arrayClone(arguments);
```

Comment:
This is a shallow clone? So it's no heavier than the fromPathsOrPathValues it's replacing?